uber/site_sections/registration.py

```diff
@@ -32,7 +32,7 @@ def checking_at_the_door(self, *args, **kwargs):
 
 @all_renderable(c.PEOPLE, c.REG_AT_CON)
 class Root:
-    def index(self, session, message='', page='0', search_text='', uploaded_id='', order='last_first', invalid=''):
+    def index(self, session, message='', page='1', search_text='', uploaded_id='', order='last_first', invalid=''):
```

Comment:
Ah, I'd misunderstood the ticket and didn't remember that we were already doing the thing I suggested.

I have no objection to implementing the change Dom says, i.e. the "always show the first page if DEV_BOX`` is true.

There's a semi-related annoyance on this page which is that if you click on one of the columns to sort by that column, it also doesn't show the first page of results by default.  Like searching, that's also a situation where we probably want to show results by default.  That doesn't need to happen as part of this PR though.